src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -106,7 +106,14 @@ def FindModuleSpace():
     if os.path.isdir(module_space):
       return module_space
 
-    runfiles_pattern = r'(.*\.runfiles)' + (r'\\' if IsWindows() else '/') + '.*'
+    slash_pattern = r'\\' if IsWindows() else '/'
+    runfiles_pattern = r'(.*\.runfiles)' + slash_pattern + '.*'
+    matchobj = re.match(runfiles_pattern, stub_filename)
+    if matchobj:
+      return matchobj.group(1)
+
+    # bazel pyz archives create a directory named "runfiles".  Try that next:
+    runfiles_pattern = r'(.*' + slash_pattern + 'runfiles)' + slash_pattern + '.*'
```

Comment:
This should be addressed in pyz archive - but doing it there will break users. So providing a second archive that's correctly packaged is probably better.

Runfiles are used by other rules as well. Fixing it in Python stub doesn't work for them.